Dashboard docs updated (#65)
Mirantis/mke-docs: content/docs/concepts/dashboard.md

---
title: MKE Dashboard
weight: 5
---

{{< callout type="info" >}} Available since 4.0.0-alpha.2.0 {{< /callout >}}

The MKE Dashboard add-on provides the web UI to manage Kubernetes resources management:

![MKE dasboard preview](ui-preview.png)

> The MKE Dashboard is enabled by default in the MKE installation.
> To access the MKE Dashboard, visit the address of the load balancer endpoint
> on a freshly-installed cluster. For details, refer to [Load balancer requirements](../../getting-started/system-requirements#load-balancer-requirements).

Reviewer: ```suggestion
To access the MKE Dashboard, which is enabled by default, navigate to the address of the load balancer endpoint
from a freshly-installed cluster. Refer to [Load balancer requirements](../../getting-started/system-requirements#load-balancer-requirements) for detailed information.
```

> To disable the MKE Dashboard, set the `enabled` field to `false` in the `dashboard`

Reviewer: ```suggestion
You can disable the MKE Dashboard by setting the `enabled` field to `false` in the `dashboard`
```
